Content.Client/LateJoin/LateJoinGui.cs

```diff
@@ -226,35 +228,39 @@ private void RebuildUI()
                         };
 
                         var specifier = new SpriteSpecifier.Rsi(new ResourcePath("/Textures/Interface/Misc/job_icons.rsi"), prototype.Icon);
-                        icon.Texture = specifier.Frame0();
+                        icon.Texture = _sprites.Frame0(specifier);
                         jobSelector.AddChild(icon);
 
                         var jobLabel = new Label
                         {
-                            Text = value != null ?
+                            Text = " " + (value != null ?
```

Comment:
is this done to have padding? it really shouldnt be done that way

Reply:
Why not? I find it much cleaner than a MinWidth Control, and I'm pretty sure it's done at some other places in the codebase.